libs/agno/agno/models/google/gemini.py

```diff
@@ -1083,3 +1094,613 @@ def _get_metrics(self, response_usage: GenerateContentResponseUsageMetadata) ->
             metrics.provider_metrics = {"traffic_type": response_usage.traffic_type}
 
         return metrics
+
+    def create_file_search_store(self, display_name: Optional[str] = None) -> Any:
+        """
+        Create a new File Search store.
+
+        Args:
+            display_name: Optional display name for the store
+
+        Returns:
+            FileSearchStore: The created File Search store object
+        """
+        config: Dict[str, Any] = {}
+        if display_name:
+            config["display_name"] = display_name
+
+        try:
+            if config:
```

Comment:
Can do it without this `if` right? Can just pass `None` to the config if not provided.